documentation/user-guide/customizing-installation.rst

```diff
@@ -336,6 +336,11 @@ For example in the job XML::
     variable if you want to supply explicit partitioning commands in some other
     way, for example in a ``<ks_append/>`` section.
 
+``no_ks_template``
+    Omits whole kickstart. Default Beaker's templating is not used.
```

Comment:
I'd rearrange things as follows:
"The whole kickstart has to be defined by user in ``<kickstart />`` tag.
Default Beaker's templating is not used; however, user does have
access to snippets and restricted context."
I didn't understand what you meant by "Omits whole kickstart." so left out.

Reply:
Fixed.